api/post.go

```diff
@@ -470,11 +473,11 @@ func getOpenGraphMetadata(c *Context, w http.ResponseWriter, r *http.Request) {
 	og := app.GetOpenGraphMetadata(url)
 
 	ogJSON, err := og.ToJSON()
+	openGraphDataCache.AddWithExpiresInSecs(props["url"], ogJSON, openGraphDataCacheExpireTime) // Cache would expire after 1 week
```

Comment:
why changing the cache time for this here in this PR, and why move it before the error?